src/components/wallet/views/loggedInContent.tsx

```diff
@@ -55,5 +55,17 @@ export default function loggedInContent(this: Wallet) {
         </pre>
       </collapsible-container>
     ) : null,
+
+    loHas(this.account, 'state') ? (
+      <collapsible-container
+        id="account-details-container"
```

Comment:
Are you sure we can have 2 `collapsible-container` components with the same id?

Reply:
Apologies still getting used css things.